__tests__/app.test.js

```diff
@@ -131,4 +131,72 @@ describe('GET api/articles', () => {
                 expect(articles[12].created_at).toBe('2020-01-07T14:08:00.000Z')
             })
     })
+});
+
+describe('GET api/articles/:article_id/comments', () => {
+    test('sending an API request to api/articles/:articles_id/comments returns a Status-200: responds with an object, with a key comments, which has the value of an array of comments relevant to given id - in this case, article id 1 should have 11 comments.', () => {
```

Comment:
quite a verbose test description - as you have an apt describe block, and you only want to focus on the behaviour tested here, you could shorten this to: Status-200: responds with an array of comments relevant to given id